src/drivers/nuopc/cime_config/config_component.xml

```diff
@@ -0,0 +1,2527 @@
+<?xml version="1.0"?>
+
+<?xml-stylesheet type="text/xsl" href="config_compsets.xsl" ?>
+
+<entry_id version="2.0">
+
+<!-- The list of component classes that this coupler/driver knows how
```

Comment:
I'm confused as to why this is appearing as a new file.

Reply:
Someone changed one line on the E3SM, probably not intentionally, and it restored the entire file. I just removed it.

Reply:
thanks - i missed this